### DISCUSSION TITLE
Fulfillment Inbound API v2024-03-20 listPackingOptions Always return only 1 option

### DISCUSSION THREAD

**Discussion by pileileishow** [amzn/selling-partner-api-models#4069] [Q&A]
Fulfillment Inbound API v2024-03-20 listPackingOptions Always return only 1 option

listPackingOptions is never returned according to the case guide. The following nodes all return empty supportedShippingModes, packingOptions->fees, and packingOptions->discounts. I tried other MSKUs and still, it almost crashed! But I continue to follow the process and return the relevant fees in listPlacementOptions and listTransportationOptions. I don't understand why this is happening and the actual expected results are not achieved.

createInboundPlan Request：
{ "destinationMarketplaces": [ "ATVPDKIKX0DER" ], "items": [ { "labelOwner": "SELLER", "msku": "12324", "prepOwner": "SELLER", "quantity": 42 } ], "name": "PLAIN1-TEST", "sourceAddress": { "addressLine1": "11111", "addressLine2": "", "city": "Jurupa Valley", "companyName": "", "countryCode": "US", "email": "", "name": "11111", "phoneNumber": "123", "postalCode": "91752", "stateOrProvinceCode": "CA" } }

generatePackingOptions Response :
{ "operation": "generatePackingOptions", "operationId": "426f7bba-d920-4fb5-a09c-bc56e5385d19", "operationProblems": [], "operationStatus": "SUCCESS" }

listPackingOptions Response:
{ "packingOptions": [ { "discounts": [], "fees": [], "packingGroups": [ "pgd7b60247-7322-4ae3-8d38-03f66ff04557" ], "packingOptionId": "po00000000-0000-0000-0000-000000000000", "status": "ACCEPTED", "supportedShippingConfigurations": [] } ], "pagination": {} }

**Comment by johnkw** [comment]
I've only ever seen 1 packingOption as well.  It's not even clear what the purpose of a packingOption is. Just select it and continue.
